ariesb/autokin
# Add: playwright support of web testing

In this PR, we

1. Added new end-to-end testing library Playwright
2. Added the capability of testing different types of browser, eg: Chromium, Firefox 

**How to test this feature:**
1. At `lib/web/autokin-web.js`, uncomment the part of playwright and comment the part of puppeteer.
2. Created the feature test case under `features/first.feature`
3. Paste the test case below:
```
@autokin 
Feature: My Feature
	As Autokin developer
	I want to verify that all API are working as they should

Scenario: My First Scenario
    Given that I use "firefox"
    Given that I have access to "someURLThatCanLogin"
    When I navigate to "/"
    Then I type "EMAIL" in field "#login-email-field"
    Then I type "PASSWORD" in field "#login-password-field"
    Then I click on "#button_login"
```

That should work and all steps should pass.

Please let me know if any help needed!
